site/en/docs/privacy-sandbox/protected-audience-api/sequential-auction-setup/index.md

```diff
@@ -0,0 +1,255 @@
+7. The Publisher Ad Server client-side library either renders the winning Protected Audience ad or contextual ad, based on what was returned from the `runAdAuction()` call. 
+8. The winning ad is rendered to the user.
+
+### Pre-auction
+
+<figure>
+  {% Img src="image/hVf1flv5Jdag8OQKYqOcJgWUvtz1/nasCbCGL8tuOeWT7VerM.png", alt="User is added to an interest group on an advertiser site", width="800", height="349" %}
+  <figcaption>
+    Interest group time sequence on an advertiser page<br/>
+    <a href="https://wd.imgix.net/image/hVf1flv5Jdag8OQKYqOcJgWUvtz1/nasCbCGL8tuOeWT7VerM.png" target="_blank">View full size diagram</a>
+  </figcaption>
+</figure>
+
+Before the auction, when the user visits an advertiser page, the buyer and the advertiser can define the site’s interest group the user belongs in, and add contextual data from the advertiser’s site and first-party data to be used as signals for the auction later. 
+
+1. The user navigates to the advertiser site.
+2. The advertiser site loads the script from each buyer participating in the auction at a later point in time. 
+3. The buyer’s script contains the `joinAdInterestGroup()` call to add the user to the buyer’s interest group. 
+
+### Contextual auctions with Prebid.js and Publisher Ad Server
+
+<figure>
+  {% Img src="image/hVf1flv5Jdag8OQKYqOcJgWUvtz1/s6skvEz9boQNm98BLtW7.png", alt="Contextual auction is initiated on the publisher site", width="800", height="349" %}
+  <figcaption>
+    Contextual auction time sequence on the publisher page<br/>
+    <a href="https://wd.imgix.net/image/hVf1flv5Jdag8OQKYqOcJgWUvtz1/s6skvEz9boQNm98BLtW7.png" target="_blank">View full size diagram</a>
+  </figcaption>
+</figure>
+
+In a sequential auction setup, all contextual auctions are executed before the Protected Audience auction runs. In the setup explained in this document, we run a header bidding contextual auction by Prebid.js that feeds into a server-side auction by the Publisher Ad Server. 
+
+The publisher first initiates a header bidding contextual auction by calling Prebid.js with [a flag](https://docs.prebid.org/dev-docs/modules/fledgeForGpt.html#:~:text=bidderRequest.fledgeEnabled) to note that a Protected Audience will be executed afterwards. Then Prebid.js [collects](https://docs.prebid.org/dev-docs/modules/fledgeForGpt.html) the bid responses and sends them to the Publisher Ad Server for a server-side contextual auction. During the bid response collection step, the buyers and sellers have the opportunity to provide component auction configs and buyers’ signals (`perBuyerSignals`) to be used for the subsequent Protected Audience auction, if they wish to participate. That component auction config will eventually pass into the subsequent Protected Audience auction.
```

Comment:
note that a Protected Audience will be executed afterwards...  protected audience what? missing word?

Reply:
good catch!  updated.